Products/CMFPlone/tests/testCatalogTool.py

```diff
@@ -547,12 +547,21 @@ def afterSetUp(self):
         self.folder.doc5.setTitle('Document 2')
         self.folder.invokeFactory('Document', id='doc6', text=RichTextValue('bar', 'text/html', 'text/x-html-safe'))
         self.folder.doc6.setTitle('DOCUMENT 4')
+
+        self.folder.invokeFactory('Document', id="doc7")
+        self.folder.doc7.setTitle('This is a long title that will ignore this text BBBB 2nd position cut zzzzz')
```

Comment:
In the new situation a title like this will only confuse, because it will no longer be ignored.  Maybe something like: "Long titles used to be truncated, but we changed this, see issue 3690", in two variants.

Reply:
@erral any chance to change this and merge it?

Reply:
I have just changed the titles in the tests.